win32/src/winapi/user32/message.rs

```diff
@@ -199,6 +199,25 @@ fn enqueue_paint_if_needed(machine: &mut Machine, hwnd: HWND) -> bool {
     true
 }
 
+fn find_message(machine: &mut Machine, hwnd: HWND, min: u32, max: u32) -> Option<usize> {
+    machine.state.user32.messages.iter().position(|msg| {
+        if !hwnd.is_null() && (!msg.hwnd.is_null() && msg.hwnd != hwnd) {
+            return false;
+        }
+        if min != 0 && max != 0 && (msg.message < min || msg.message > max) {
+            return false;
+        }
+        true
+    })
+}
+
+fn copy_message(dstMsg: &mut MSG, srcMsg: &MSG) {
+    // Old versions of MSG don't have the lPrivate field. It might point to another stack variable, so don't overwrite it.
```

Comment:
Oh one other thing I was thinking about here is maybe we should just drop lPrivate from the retrowin32 struct, what do you think?  I imagine you figured this bit out because retrowin32 broke something.

Reply:
I can’t find any documentation that shows what `lPrivate` is used for. I noticed there’s no `lPrivate` field in Wine, so maybe it can be omitted from the retrowin32 struct as well.